src/EoS/Nuclear/NuclearEoS.cpp

```diff
@@ -163,12 +270,11 @@ void nuc_eos_C_short( const real xrho, real *xenr, const real xye,
       {
          const real lprs = LOG10( *xprs );
 
-         if ( lprs > logprss_mode[nmode-1] ) {  *keyerr = 111;  return;  }
-         if ( lprs < logprss_mode[      0] ) {  *keyerr = 112;  return;  }
-         if ( lprs != lprs )                 {  *keyerr = 206;  return;  }
+         if ( lprs > logprss_mode[nmode-1] ) {  *keyerr = 113;  return;  }
+         if ( lprs < logprss_mode[      0] ) {  *keyerr = 114;  return;  }
 
          findenergy( lr, lprs, xye, &leps, alltables_mode, nrho, nmode, nye, neps,
```

Comment:
I think we should use ``nrho_mode` and `nye_mode` here, right?

Reply:
you are right.